README.md

```diff
@@ -303,9 +303,9 @@ $ biko gcp [product] [flag(s)]
 | App Engine | Go to GAE Dashboard | `appengine`, `gae` | - |
 | Compute Engine | Go to GCE page | `compute` | -  |
 | Cloud Functions | Go to Cloud Functions page or the functions detail | `functions`, `f` |  `--region, -r`, `--name, -n` |
-| Cloud Run | Go to Cloud Run page or the deployments detail | `run` | `--region, -r`, `--name, -n` |	
+| Cloud Run | Go to Cloud Run page or the deployments detail | `run` | `--region, -r`, `--name, -n` |
 | KMS | Go to security's cryptographic keys page | `kms` | - |
-| Kubernetes | Go to GKE page, or the cluster detail | `kubernetes`, `gke` | `--region, -r`, `--name, -n` | 
+| Kubernetes | Go to GKE page, or the cluster detail | `kubernetes`, `gke` | `--region, -r`, `--name, -n` `--namespace, -ns` |
```

Comment:
```suggestion
| Kubernetes | Go to GKE page, or the cluster detail | `kubernetes`, `gke` | `--region, -r`, `--name, -n` `--namespaces, -ns` |
```

Reply:
The flag description on the doc should be `--namespaces` 🙇

Reply:
Oh! Sorry for my missing it. Fixed!

Reply:
Solved in https://github.com/KeisukeYamashita/biko/pull/105/commits/13ae27bd7f37c340cca7bc0da10fb183d4fd24fd, thanks!